CHANGELOG.md

```diff
@@ -65,4 +77,4 @@ Initial release of MLAPI Bitesize Samples repository. Samples support the follow
 
 ### License
 
-The Bitesize Samples repository and code projects are licensed under the Unity Companion License for Unity-dependent projects (see https://unity3d.com/legal/licenses/unity_companion_license). See [LICENSE](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/LICENSE.md) for full details.
\ No newline at end of file
+The Bitesize Samples repository and code projects are licensed under the Unity Companion License for Unity-dependent projects (see https://unity3d.com/legal/licenses/unity_companion_license). See [LICENSE](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/LICENSE.md) for full details.
```

Comment:
```suggestion
The Bitesize Samples repository and code projects are licensed under the Unity Companion License for Unity-dependent projects (see https://unity3d.com/legal/licenses/unity_companion_license). See [LICENSE](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/main/LICENSE.md) for full details.
```
Don't think anything changed here but taking a chance to get rid of another `master`